.goreleaser.yaml

```
@@ -7,6 +7,20 @@ builds:
       - windows
       - darwin
     binary: tagger
+dockers:
+  # build latest and specific tag version images
+  - image_templates:
+      - "linode/{{.ProjectName}}:{{ .Tag }}"
+      - "linode/{{.ProjectName}}:latest"
+      # push to github container registry too
+      - "ghcr.io/linode/{{.ProjectName}}:{{ .Tag }}"
+      - "ghcr.io/linode/{{.ProjectName}}:latest"
```

I'm not sure I can push to Linode's official GHCR account from a project on my account like this (it requires GH tokens for the right account). Besides, this isn't an official tool right now. For the time being, I think these images should be pushed to my account.
```suggestion
      - "tjhop/{{.ProjectName}}:{{ .Tag }}"
      - "tjhop/{{.ProjectName}}:latest"
      # push to github container registry too
      - "ghcr.io/tjhop/{{.ProjectName}}:{{ .Tag }}"
      - "ghcr.io/tjhop/{{.ProjectName}}:latest"
```